lib/checks/text-alternatives/link-text.js

```diff
@@ -0,0 +1,12 @@
+let nodeText = axe.commons.text.accessibleText(node);
+let nodeDescripbedBy = axe.commons.aria.ariaDescribedby(node);
```

Comment:
nitpick, spelling mistake - `nodeDescibedBy`

Reply:
This still needs a tweak to be spelled correctly: `nodeDescribedBy` (missing the `r`)